internal/repository/http.go

```diff
@@ -89,7 +89,7 @@ func (r *HTTPRepository) Exists(u fyne.URI) (bool, error) {
 // Reader provides a interface for reading the body of the response received
 // from the request to u.
 //
-// The method is a part of the implementation for repository.Repository.Exists
```

Comment:
This doesn't seem right - these items are not headings.
Occurs in a couple of places - just noting it here

Reply:
Good point. I sorted out the issue and also fixed a comment that accidentally said "Sine:" instead of "Since:" (we don't want it to periodically be in releases, hehe).